server/server.go

```diff
@@ -469,7 +469,7 @@ func (s *Server) Flush(ctx context.Context, req *spb.FlushRequest) (*spb.FlushRe
 		// Always use codes.Internal here since any error (not being able to flush, or
 		// not finding an NI that we already checked existed is some internal logic
 		// error).
-		return nil, status.Errorf(codes.Internal, det.String())
+		return nil, status.Errorf(codes.Internal, "%s", det.String())
```

Comment:
the function signature should also be updated to show returning three values?

Reply:
We're still returning two here -- this was just addressing a linter error where `det.String()` was being considered as a formatting string. `status.Errorf` returns exactly one `error`.